intellij_platform_sdk/BUILD.clion243

```diff
@@ -59,6 +59,8 @@ java_import(
         "plugins/clion-test-google/lib/*.jar",
         "plugins/clion-test-catch/lib/*.jar",
         "plugins/clion-test-boost/lib/*.jar",
+        "plugins/clion-radler/lib/*.jar",
```

Comment:
a possible solution: define this in a separate `java_import` statement, use that library in https://github.com/bazelbuild/intellij/pull/7062/files#diff-242eb0d3dc78241907aa160705183dba65b9f3cb747ee146887eaf081f7622f5R18 and let's keep using classic so far for development purposes

Reply:
I played around with multiple options, I think just using the suppressed plugin set options is the easiest for now. This allows us later to run test for classic and radler. Tho radler requires the path to the backend binary which I could not get working for now.